ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md

```diff
@@ -1,79 +1,118 @@
 # Validator Setup
-## Requirements 
+
+- [Validator Setup](#validator-setup)
+  - [Requirements](#requirements)
+  - [Firewall](#firewall)
+    - [Validator](#validator)
+    - [VFN](#vfn)
+  - [High-level steps](#high-level-steps)
+  - [TMUX basics](#tmux-basics)
+  - [1. Set up a host](#1-set-up-a-host)
+    - [Create Binaries](#create-binaries)
+  - [2. Generate account keys](#2-generate-account-keys)
+  - [3. Fast forward to the most recent state snapshot](#3-fast-forward-to-the-most-recent-state-snapshot)
+  - [4. Start the validator in `fullnode` mode](#4-start-the-validator-in-fullnode-mode)
+  - [5. Start producing delay proofs on validator ("delay mining")](#5-start-producing-delay-proofs-on-validator-delay-mining)
+  - [6. Create VFN config and deploy the VFN](#6-create-vfn-config-and-deploy-the-vfn)
+    - [Return to validator machine](#return-to-validator-machine)
+    - [Return to VFN](#return-to-vfn)
+  - [7. Restart validator node in `validator` mode](#7-restart-validator-node-in-validator-mode)
+  - [[Optional] Establish metric exporter for centralized monitoring](#optional-establish-metric-exporter-for-centralized-monitoring)
+  - [Onboarder Troubleshooting](#onboarder-troubleshooting)
+  - [Onboarder instructions](#onboarder-instructions)
+  - [Troubleshooting](#troubleshooting)
+    - [cargo or rust are not installed](#cargo-or-rust-are-not-installed)
+
+## Requirements
+
 - TWO unix hosts, one for Validator Node, and one for the Private Fullnode ("VFN").
 0L code targets Ubuntu 20.4
-- Recommended specs: 
-  - Validator: 250G harddrive, 8 core CPU, 16G RaM
+- Recommended specs:
+  - Validator: 250G harddrive, 8 core CPU, 16G RAM
```

Comment:
Let's bump the hard drive requirement to 300GB SSD.

Reply:
modified